pkg/quarterdeck/orgs.go

```diff
@@ -233,20 +234,20 @@ func (s *Server) ProjectAccess(c *gin.Context) {
 	// Fetch the user claims from the request
 	if claims, err = middleware.GetClaims(c); err != nil {
 		sentry.Error(c).Err(err).Msg("could not get user claims from authenticated request")
-		c.JSON(http.StatusUnauthorized, api.ErrorResponse(api.ErrInvalidUserClaims))
+		c.JSON(http.StatusUnauthorized, api.ErrorResponse(responses.ErrNeedPermission))
```

Comment:
Same here as the comment above.

Reply:
Same question as above - if the failure is due to something internal that Ensign has done wrong, should the error be `StatusInternalServerError` instead of `StatusUnauthorized`?